From f66688ab4ff3de610dba7f0f91ecd202ab48bf3c Mon Sep 17 00:00:00 2001
From: Robert Farmer <rjfarmer@asu.edu>
Date: Wed, 23 Aug 2017 14:53:51 +0100
Subject: [PATCH 4/4] sdk with lapack

---
 install               |  2 +-
 utils/makefile_header | 10 +++++-----
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/install b/install
index 204dac8..f0c1842 100755
--- a/install
+++ b/install
@@ -244,7 +244,7 @@ do_one star
 
 do_one_parallel binary
 
-#do_one gyre
+do_one gyre
 
 #do_one adipls
 
diff --git a/utils/makefile_header b/utils/makefile_header
index 54e6698..6afa2db 100644
--- a/utils/makefile_header
+++ b/utils/makefile_header
@@ -50,10 +50,10 @@ SPECIAL_C_FLAGS  = -fPIC -shared
 
 # step 3) specify which BLAS and LAPACK libraries to use for mesa/mtx
 
-WHICH_LAPACK = USE_SRCS
-LOAD_LAPACK = -lmesalapack
-WHICH_BLAS = USE_SRCS
-LOAD_BLAS = -lmesablas
+WHICH_LAPACK = 
+LOAD_LAPACK = `mesasdk_lapack_link`
+WHICH_BLAS = 
+LOAD_BLAS = `mesasdk_blas_link`
 
 
 # step 4) do you want to use PGPLOT with mesa/star?                                                                                     
@@ -74,7 +74,7 @@ INCLUDE_HDF5 = -I${MESASDK_ROOT}/include
 # if you don't plan to use GYRE and want to skip building it, then
 #USE_GYRE = NO
 # if you want to have GYRE built along with the rest of mesa, then
-USE_GYRE = NO
+USE_GYRE = YES
 
 
 # step 7) now do the mesa install
-- 
2.9.5

